maintaining-and-tuning/monitoring.md

```diff
@@ -155,38 +172,42 @@ access to the port, for example by employing a firewall. Furthermore, read
 [Prometheus Security](https://prometheus.io/docs/operating/security/).
 
 ## Example visualization and alerting setup
-This example will walk you through setting up Grafana as a dedicated visualization software to parse, display and query metrics scraped from GRR server components by Prometheus. You will also be able to set up a simple alerting system using Grafana.
-These instructions assume that GRR server and Prometheus are both up and running.
+This example will walk you through setting up Grafana as a dedicated visualization software to parse, display and query metrics scraped from GRR server components by Prometheus. If Prometheus was configured to scrape Fleetspeak servers as well, all steps apply. You will also be able to set up a simple alerting system using Grafana.
```

Comment:
>  If Prometheus was configured to scrape Fleetspeak servers as well, all steps apply. 

And if it wasn't? It's not clear from this sentence, which steps do/don't apply then.

Reply:
You're right; Prometheus would have been configured to scrape Fleetspeak back in the example setup from the top of the doc. I'll link it for clarity.